login.html

```diff
@@ -12,15 +12,18 @@
     <div class="background-login">
       <div class="container-login">
         <img class="login-logo" src="./assets/LogoWolox.png" alt="Wolox logo">
-        <label for="mail">Email</label>
-        <input type="email" name="mail">
-        <label for="pass">Password</label>
-        <input type="password" name="pass">
-        <a href="list.html">
-          <button class="btn-green">Login</button>
-        </a>
-        <hr>
-        <button class="btn-inverted-green">Sing up</button>
+        <div class="form-group">
```

Comment:
Do you really need this div?

Reply:
is the way I found to put a line between the login and register buttons after removing the hr